lib/plugins/extension/lang/de/lang.php

```diff
@@ -46,7 +47,7 @@
 $lang['tags']                  = 'Schlagworte';
 $lang['author_hint']           = 'Suche weitere Erweiterungen dieses Autors';
 $lang['installed']             = 'Installiert:';
-$lang['downloadurl']           = 'URL zum Herunterladen:';
+$lang['downloadurl']           = 'Download-URL:';
 $lang['repository']            = 'Quelle:';
 $lang['unknown']               = '<em>unbekannt</em>';
 $lang['installed_version']     = 'Installierte Version:';
```

Comment:
Need some help from German speakers: is this "Download-URL" German?

Reply:
Yes, it is.

Reply:
Thanks!